CommandHelper/LocalPackages/commands/chars/vegetasupersaiyanbeyondblue

```diff
@@ -0,0 +1,40 @@
+alts:
```

Comment:
file name should be vegetasupersaiyanbeyondblue.yml not vegetasupersaiyanbeyondblue